chia/full_node/lock_queue.py

```diff
@@ -54,18 +46,26 @@ class LockQueue(Generic[_T_Comparable]):
     Must be created while an asyncio loop is running.
     """
 
-    _queue: queue.PriorityQueue[_Element[_T_Comparable]] = dataclasses.field(default_factory=queue.PriorityQueue)
-    _active: Optional[_Element[_T_Comparable]] = None
-    _creation_counter: int = 0
-    cancelled: Set[_Element[_T_Comparable]] = dataclasses.field(default_factory=set)
+    _deques: Dict[int, collections.deque[_Element]]
+    _priority_index_map: Dict[_T_Priority, int]
+    _priority_type: Type[_T_Priority]
+    _active: Optional[_Element] = None
 
     # TODO: can we catch all unhandled errors and mark ourselves broken?
     # TODO: add debug logging
 
+    @classmethod
+    async def create(cls, priority_type: Type[_T_Priority]) -> LockQueue[_T_Priority]:
```

Comment:
Right, it probably doesn't.  I intentionally made it so because we used to have to create this when there was a loop due to the `asyncio.PriorityQueue` and making this (otherwise pointlessly) async encouraged that use.  But...  not needed now (if I remember correctly).